apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -190,48 +190,33 @@ You now have a block package that you can provide to apps to use, by **publishin
 
 ## Publish
 
-Once you've built a block, you can add it to the [Hub](/hub), so that:
-
-- your block will have an instant online demo playground
-- your block will be searchable via our block API
-- you can claim your namespace in blockprotocol.org
-
-To publish a block on the Hub, the initial process is as follows:
-
-1.  Have your block source code available in a GitHub repository
-1.  Claim your username by signing up at [https://blockprotocol.org](/)
-1.  Fork the [`blockprotocol` repository](https://github.com/blockprotocol/blockprotocol)
-1.  Add a JSON file to the [hub folder](https://github.com/blockprotocol/blockprotocol/tree/main/hub), in a subfolder matching the username you registered in step one, preceded by the `@` symbol (e.g. `@myusername`)
-1.  [Create a pull request](https://github.com/blockprotocol/blockprotocol/compare) from your fork
-
-The JSON metadata file should look like this:
-
-```json5
-{
-  // REQUIRED - path to the repository with your block's source code
-  repository: "https://github.com/hashintel/hash.git",
-  // REQUIRED - commit hash (pins block to a specific version)
-  commit: "adb915bf8fc4f84e33a2cd21f217225e50c3d7fa",
-
-  // IF NEEDED - see below
-  distDir: "dist",
-  folder: "blocks/block-embed",
-  workspace: "@hashintel/block-embed",
-}
-```
+Once you've built a block, you can add it to the [Hub](/hub), so that your block will have an instant online demo playground,
+and will be searchable via our block API.
+
+To publish a block on the Hub take the following steps.
+
+### In your browser
 
-### My block source is in a subfolder
+- Sign up for an account on this site if you haven’t already – click 'sign up' in the top right
+- [Create an API key](https://blockprotocol.org/settings/api-keys and copy it
```

Comment:
```suggestion
- [Create an API key](https://blockprotocol.org/settings/api-keys) and copy it
```

```suggestion
- [Create an API key](/settings/api-keys) and copy it
```

The second option aligns the link with `[Hub](/hub)` above. You may want to use absolute links everywhere instead.

Reply:
I've gone for absolute links everywhere in https://github.com/blockprotocol/blockprotocol/pull/713/commits/d22ea1720e6eeacd4b164b3a5df46e233d478c51

I have in my head that this is better because it allows the links to work wherever the markdown is being read. It has no effect in development since links starting with `https://blockprotocol.org` are [treated as internal](https://github.com/blockprotocol/blockprotocol/blob/4bf770dd3c2f854f2109f59b0043137475c63d91/apps/site/src/components/base-link.tsx#L18) and will be routed accordingly.

Reply:
Thanks, makes total sense. If we need to fix URLs in preview deployments, I guess we can do that in a separate PR by tweaking markdown AST.